app/bibleview-js/src/components/BibleView.vue

```diff
@@ -417,4 +417,9 @@ a {
   text-decoration: none;
   display: inline-block;
 }
+.strongs {
+  font-size: 0.6em;
+  text-decoration: none;
+  color: coral;
+}
```

Comment:
These lines should be in W.vue

Reply:
Ok, I have moved those changes across. So now i think it is correct. Yes?